sql/src/main/java/io/crate/planner/PlanNodeBuilder.java

```diff
@@ -53,12 +58,26 @@ public static CollectNode distributingCollect(TableInfo tableInfo,
         node.downStreamNodes(downstreamNodes);
         node.toCollect(toCollect);
         node.projections(projections);
-
         node.isPartitioned(tableInfo.isPartitioned());
         setOutputTypes(node);
         return node;
     }
 
+    public static CollectNode distributingCollect(TableInfo tableInfo,
+                                                  WhereClause whereClause,
+                                                  List<Symbol> toCollect,
+                                                  List<String> downstreamNodes,
+                                                  ImmutableList<Projection> projections,
+                                                  @Nullable OrderBy orderBy,
+                                                  @Nullable Integer limit) {
+        CollectNode node = distributingCollect(tableInfo, whereClause, toCollect, downstreamNodes, projections);
```

Comment:
can you inline the stuff that happens in distributingCollect and remove the second function?